README.md

```diff
@@ -99,6 +99,15 @@ For more information on API usage, please refer to [src/native/api.h](https://gi
 * Python: [pallas/pyllhttp][8]
 * Ruby: [metabahn/llhttp][9]
 
+## Generating on Windows
+
+* choco install git
+* choco install node
+* choco install llvm
```

Comment:
Do you still need `llvm` for building `llhttp`? I think this shouldn't be a requirement anymore.

Reply:
lol. I'm actually having issues linking the object file under Windows. I created an issue [here](https://github.com/nodejs/llhttp/issues/109).
If I can find a way to get this issue resolved I update these instructions with more detail.

Reply:
It likely will work with other compilers. I've just tried with clang and it worked.